src/handbook/peopleops/hiring.md

```diff
@@ -79,30 +72,28 @@ During the onboarding process on our EOR provider platform, candidates must eith
 
 ## Onboarding
 
-Just before your first day of work at FlowForge, you will receive an email requesting your GitHub username and preferred company email address. An issue
-will be created on our [internal issue tracker][issue-tracker] with steps for you and
+Just before your first day of work at FlowForge, you will receive an email requesting your GitHub username and preferred company email address. An issue will be created on our [internal issue tracker][issue-tracker] with steps for you and
 the team to complete once you start.
 
-On the day you start, a message will be sent to your personal email address to grant you access to your FlowForge email address.
+On the day of your start, a message will be sent to your personal email address, providing you access to your FlowForge email address. This process ensures that all new joiners receive access either on their first day or up to 24 hours before their official start date.
 
-After setting up your password, you must [turn on the 2-Step Verification](https://support.google.com/accounts/answer/185839?hl=en&co=GENIE.Platform%3DDesktop). Otherwise, after a few days, you will be locked out of your account and will need an admin's help to log back in.
+Upon setting up your password, it is essential to [turn on the 2-Step Verification](https://support.google.com/accounts/answer/185839?hl=en&co=GENIE.Platform%3DDesktop). Failure to do so may result in being locked out of your account after a few days, requiring assistance from an admin to regain access.
 
 Once your FlowForge email is set up, you will have access to invitations to other services and platforms, including 1Password. [Set this up](https://support.1password.com/explore/team-member/) as soon as possible to add a security layer to all your FlowForge accounts.
 
-On your first day, after your email and 1Password accounts are set up, you should prioritize gaining access to Slack, the FlowForge GitHub organization, and completing your onboarding issue. Your manager will also assign work for you to start on in the first week.
-
-[issue-tracker]: https://github.com/flowforge/admin/issues
+On your first day, once your email and 1Password accounts are set up, it is important to prioritize gaining access to Slack, the FlowForge GitHub organization, and completing your onboarding issue. Additionally, your manager will assign tasks for you to begin working on during your first week.
 
 ## Greenhouse
 
 ### Opening a Job
 
 When opening a job post, you'll need a couple of things:
+
 1. A draft Job Description
-1. Approved role availability
-1. Support for a PeopleOps Manager to setup the role in Greenhouse
+2. Verifying the availability of the approved role.
+3. The PeopleOps Manager will collaborate with the hiring manager to set up the role within Greenhouse. They will provide support in case any custom alterations to the job post are required.
```

Comment:
```suggestion
3. Set up the role within Greenhouse. The Peopleops manager will provide support in case any custom alterations to the job post are required.
```